.github/workflows/build.yml

```diff
@@ -114,7 +114,9 @@ jobs:
       - uses: ./.github/actions/setup
       - name: build test container
         run: |
-          podman build --squash --tag test --build-arg base="$(./build --print-container-image)" tests
+          OCI_IMAGE_TAG="$(./build --print-container-image)"
```

Comment:
This will always print `localhost/builder`, but I think it's better not to have that string hard coded here, so it will also work in other cases.